src/components/Posts/SinglePost.jsx

```diff
@@ -19,38 +28,76 @@ export function SinglePost({ article, id }) {
     hour: "numeric",
     minute: "numeric",
   };
+
   useEffect(() => {
     (async function getComments() {
       const articleRef = db.collection("articles").doc(id);
-      const commentsRef = articleRef.collection("comments").orderBy("timestamp", "asc");
+      const commentsRef = articleRef
+        .collection("comments")
+        .orderBy("timestamp", "desc");
```

Comment:
please make the timestamp "asc". For now, we want new comments to be added to the bottom.